src/v2/Components/FlashBanner/__tests__/FlashBanner.jest.tsx

```diff
@@ -1,22 +1,62 @@
-import { mount } from "enzyme"
+import { ReactWrapper, mount } from "enzyme"
 import "jest-styled-components"
 import React from "react"
-import { FlashBanner } from "v2/Components/FlashBanner"
+import { BaseFlashBanner } from "v2/Components/FlashBanner"
+import { renderRelayTree } from "v2/DevTools"
+import { FlashBannerTestQueryRawResponse } from "v2/__generated__/FlashBannerTestQuery.graphql"
+import { graphql } from "react-relay"
+import { Banner } from "@artsy/palette"
+
+jest.unmock("react-tracking")
+jest.unmock("react-relay")
 
 const { location: originalLocation } = window
 
-describe("FlashBanner", () => {
-  afterEach(() => {
-    window.location = originalLocation
+const mockMe: FlashBannerTestQueryRawResponse["me"] = {
+  id: "fff",
+  canRequestEmailConfirmation: false,
+}
+
+let wrapper: ReactWrapper
+
+async function getWrapper(
+  mockData: FlashBannerTestQueryRawResponse,
+  queryString: string = ""
+) {
+  delete window.location
+  const fakeLocation: any = { search: queryString }
+  window.location = fakeLocation
+
+  wrapper = await renderRelayTree({
+    Component: BaseFlashBanner,
+    mockData,
+    query: graphql`
+      query FlashBannerTestQuery @raw_response_type {
+        me {
+          canRequestEmailConfirmation
```

Comment:
Not sure of the best way to test this component since i'm skipping the QueryRenderer component and basically mounting the base component under my own test query renderer. I might have come to it a different way had I not begun with a fragment container (used on line 31 above), but simply replacing the query/fragment on L36 and component on L31 quickly made tests work after removing the fragment container (1b4c1d32ee3c2049fd5396535834e14f04262b55), so I kept that for now.